### ISSUE TITLE
Compile fails when file name contains a space

### ISSUE THREAD

**Issue by cwieland** [chunkypixel/atari-dev-studio#30]
Hi.  I have the version 0.4.9 of the extension installed on Windows 10 and Ubuntu.  When I try to compile an assembly source file that contains a space character in the file name, DASM is stopping with an exit code 1.  I believe that some of the DASM parameters just need to be wrapped in quotes.  Below are the contents from the OUTPUT tab.

```
Starting build of Scrolling Playfield - Left.asm...
DASM 2.20.13
Copyright (c) 1988-2020 by the DASM team.
License GPLv2+: GNU GPL version 2 or later (see file LICENSE).
DASM is free software: you are free to change and redistribute it.
There is ABSOLUTELY NO WARRANTY, to the extent permitted by law.

Usage: dasm sourcefile [options]

-f#      output format 1-3 (default 1)
-oname   output file name (else a.out)
-lname   list file name (else none generated)
-Lname   list file, containing all passes
-sname   symbol dump file name (else none generated)
-v#      verboseness 0-4 (default 0)
-d       debug mode (for developers)
-Dsymbol              define symbol, set to 0
-Dsymbol=expression   define symbol, set to expression
-Msymbol=expression   define symbol using EQM (same as -D)
-Idir    search directory for INCLUDE and INCBIN
-p#      maximum number of passes
-P#      maximum number of passes, with fewer checks
-T#      symbol table sorting (default 0 = alphabetical, 1 = address/value)
-E#      error format (default 0 = MS, 1 = Dillon, 2 = GNU)
-S       strict syntax checking

Report bugs on https://github.com/dasm-assembler/dasm please!


Fatal assembly error: Check command-line format.
Exit code: 1
```

**Comment by chunkypixel** [comment]
Hi Charles - thought I'd got all of these in the last pass!  Will find the issue and get this into the next release.

Thanks again for using Atari Dev Studio and logging this issue.

**Comment by chunkypixel** [comment]
Hi Charles - apologies for not getting to this sooner.  I'm waiting on an answer from the dasm guys as to how/if this can work.  I've tried a few things to no avail...

**Comment by chunkypixel** [comment]
Fixed in v0.5.1 
Will release tomorrow
